spec/classes/puppet_agent_spec.rb

```diff
@@ -52,7 +52,7 @@
         end
 
         it do
-          should contain_puppet__config__agent('server').with_value('foo.example.com')
+          should contain_puppet__config__main('server').with_value('foo.example.com')
```

Comment:
Sure? Shouldn't this be tested for the agent as well? Perhaps just a copy in puppet_config_spec.rb?